reactjs/.eslintrc.json

```diff
@@ -6,7 +6,6 @@
     },
     "extends": [
         "standard",
-        "eslint:recommended",
```

Comment:
why is this removed?

Reply:
it is obsolete, rules are completely covered with "plugin:react/recommended" extension